.eslintrc.js

```diff
@@ -17,9 +17,10 @@ module.exports = {
   'rules': {
     "require-jsdoc" : 0,
     "valid-jsdoc" : 0,
-    "max-len" : ["error", { "code": 120 ,"ignoreTemplateLiterals": true},],
+    "max-len" : ["error", { "code": 178 ,"ignoreTemplateLiterals": true},],
```

Comment:
Is there a way to assemble the command while following the max len rule?

Reply:
i did the suggested changes below